go/vt/topo/etcd2topo/server_test.go

```diff
@@ -89,6 +95,122 @@ func startEtcd(t *testing.T) (*exec.Cmd, string, string) {
 	return cmd, dataDir, clientAddr
 }
 
+// startEtcdWithTLS starts an etcd subprocess with TLS setup, and waits for it to be ready.
+func startEtcdWithTLS(t *testing.T) (*exec.Cmd, string, string) {
+	// Create a temporary directory.
+	dataDir, err := ioutil.TempDir("", "etcd")
+	if err != nil {
+		t.Fatalf("cannot create tempdir: %v", err)
+	}
+
+	// Get our two ports to listen to.
+	port := testfiles.GoVtTopoEtcd2topoPort
+	name := "vitess_unit_test"
+	clientAddr := fmt.Sprintf("http://localhost:%v", port)
```

Comment:
Theses two needs to be "https://"

Reply:
Updated: https://github.com/vitessio/vitess/pull/5496/commits/1c5e2369dc6bb986f23a91dcd8de7b33509cb8cd.